addons/web/static/src/views/debug_items.js

```diff
@@ -188,6 +188,43 @@ export function viewMetadata({ component, env }) {
 }
 
 debugRegistry.category("form").add("viewMetadata", viewMetadata);
+// -----------------------------------------------------------------------------
+// View Raw Record Data
+// -----------------------------------------------------------------------------
+class RawRecordDialog extends Component {}
+RawRecordDialog.template = xml`
+<Dialog title="props.title">
+    <pre t-esc="props.raw_data"/>
+</Dialog>`;
+RawRecordDialog.components = { Dialog };
+RawRecordDialog.props = {
+    raw_data: { type: String },
+    title: { type: String },
+    close: { type: Function },
+};
+
+export function viewRawRecord({ component, env }) {
+    const { resId, resModel } = component.model.config;
+    if (!resId) {
+        return null;
+    }
+    const description = _t("View Raw Record Data");
+    return {
+        type: "item",
+        description,
+        callback: async () => {
+            const raw_data = (await component.model.orm.read(resModel, [resId]))[0];
+            const dialogTitle = _t("Raw Record Data: %s(%s)", resModel, resId);
+            env.services.dialog.add(RawRecordDialog, {
+                title: dialogTitle,
+                raw_data: JSON.stringify(raw_data, Object.keys(raw_data).sort(), 2),
+            });
+        },
+        sequence: 325,
+    };
+}
+
+debugRegistry.category("form").add("viewRawRecord", viewRawRecord);
```

Comment:
it isn't related to this task, but I see in this file pieces of backward compatibility code (grep legacy). Could you remove it in another PR?